src/App.jsx

```diff
@@ -8,13 +8,18 @@ import { useState } from 'react'
 
 
 function App() {
- const [newTodo, setNewTodo] = useState('Activities List:');
+ const [todoList, setTodoList] = useState([]);
+
+ function addTodo(title){
+  const newTodo =  { title, id: Date.now()};
+ setTodoList([...todoList, newTodo]);
+}
```

Comment:
Be mindful of your formatting. If you have time, I would encourage you to go back to the first lesson and get Prettier working. It takes care of this for you. The 1:1 mentors can help out with this too.